### PR TITLE
Solution scenariorun ids

### PR DESCRIPTION
**Pull request by icreated**

### CODE REVIEW

File: scenariorun/src/main/kotlin/com/cosmotech/scenariorun/repository/ScenarioRunRepository.kt [hunk 1]
```diff
@@ -5,6 +5,7 @@ package com.cosmotech.scenariorun.repository
 import com.cosmotech.scenariorun.domain.ScenarioRun
 import com.redis.om.spring.annotations.Query
 import com.redis.om.spring.repository.RedisDocumentRepository
+import java.util.*
```
Comment (Leopold-Cramer) [inline scenariorun/src/main/kotlin/com/cosmotech/scenariorun/repository/ScenarioRunRepository.kt:8]: Try to not put * imports

File: scenariorun/src/test/kotlin/com/cosmotech/scenariorun/service/ScenarioRunServiceImplTests.kt [hunk 2]
```diff
@@ -34,14 +34,14 @@ import io.mockk.mockkStatic
 import io.mockk.spyk
 import io.mockk.unmockkStatic
 import io.mockk.verify
+import java.util.*
```
Comment (Leopold-Cramer) [inline scenariorun/src/test/kotlin/com/cosmotech/scenariorun/service/ScenarioRunServiceImplTests.kt:37]: Same as earlier

File: solution/src/main/kotlin/com/cosmotech/solution/repository/SolutionRepository.kt [hunk 3]
```diff
@@ -3,13 +3,22 @@
 package com.cosmotech.solution.repository
 
 import com.cosmotech.solution.domain.Solution
+import com.redis.om.spring.annotations.Query
 import com.redis.om.spring.repository.RedisDocumentRepository
+import java.util.*
```
Comment (Leopold-Cramer) [inline solution/src/main/kotlin/com/cosmotech/solution/repository/SolutionRepository.kt:8]: Same as earlier

**Review by Leopold-Cramer — APPROVED** [review]

**Review by icreated — COMMENTED** [review]
All fixed

**Pull request merged**